tutorials/applications/Simulating Sequences with Errors and Noise.ipynb

```diff
@@ -0,0 +1,899 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    pulse = Pulse.ConstantPulse(duration, 1.5*2*np.pi, 0.*2*np.pi, 0)
</pre><p>Why is there a 1.5 in front of 2pi ? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/161/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>